discovery-infra/update_bm_inventory_cm.py

```diff
@@ -36,8 +36,11 @@ def get_relevant_envs():
     data = {}
     for env in ENVS:
         evn_data = os.getenv(env[0], env[1])
-        if evn_data:
-            data[env[0]] = os.getenv(env[0], env[1])
+        # Set value as empty if variable is an empty string (e.g. defaulted in Makefile)
+        if evn_data == '""':
```

Comment:
it is not good. Why u need to set it? if not set just don't set it and inventory will use it's default

Reply:
The env var is already set in the Makefile, so os.getenv gets the value.